src/main/pg/piniobox.c

```diff
@@ -46,6 +46,22 @@
 #define PINIO4_BOX PERMANENT_ID_NONE
```

Comment:
Does any target need default for so many boxes? Maybe 5+ can be hardcoded to PERMANENT_ID_NONE?